iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -1063,8 +1064,9 @@ static unsigned decideFusableLinalgOps(FuncOp funcOp) {
             consumer.getNumLoops() != consumer.getNumParallelLoops())
           continue;
         AffineMap consumerIndexingMap =
-            consumer.getInputIndexingMap(use.getOperandNumber());
-        AffineMap producerIndexingMap = linalgOp.getOutputIndexingMap(0);
+            consumer.getIndexingMaps()[use.getOperandNumber()];
```

Comment:
getTiedIndexingMap(&use) should work as well?

Reply:
Done.